### PR TITLE
Adds support for copy/paste

### PR DESCRIPTION
**Pull request by lukaszlenart**
User can paste a name of a suggestion she's looking for

### CODE REVIEW

**Comment by arigoldx** [comment]
In terms of testing, unit tests pass & I'll give this a whirl in the [mercury PR](https://github.com/elemica/mercury/pull/8632). 

'coupla lil questions for curiosity's sakes:

What happened that we need to add the left and right arrows to the non printable keys?

aaand,

What was the trick with all that version-number craziness?

(thanks ahead of time for the lesson :))

**Comment by lukaszlenart** [comment]
> re: left & right

each time when you were navigating throughout the text (going left or right), a request was sent back to the server to fetch possible suggestions, I assumed that this wasn't an expected behaviour.

> re: version carziness

just to have a green build on Travis 😆

**Comment by arigoldx** [comment]
The complied JavaScript worked like a charm in [the mercury PR](https://github.com/elemica/mercury/pull/8632) so I'm all sorts of 👍 

@pdyraga for GK?

**Comment by arigoldx** [comment]
Noooooow I get it.. 'just hit me that the way to check the Travis build is to just keep committing until its green 😆

File: dist/elemica-suggest.js [hunk 1]
```diff
@@ -6,18 +6,22 @@ elemicaSuggest 0.9.2-SNAPSHOT
  */
 
 (function() {
+  var indexOf = [].indexOf || function(item) { for (var i = 0, l = this.length; i < l; i++) { if (i in this && this[i] === item) return i; } return -1; };
```
Comment (pdyraga) [inline dist/elemica-suggest.js]: Why do we need to define `indexOf` here? Starting from IE9 it should be available in `Array`: https://www.w3schools.com/jsref/jsref_indexof_array.asp

Is it for IE8 compatibility? I think we can drop a support for this version in the `elemica-suggest`.

File: dist/elemica-suggest.js [hunk 2]
```diff
@@ -27,6 +31,7 @@ elemicaSuggest 0.9.2-SNAPSHOT
         ALT = 18;
         OS_LEFT = 91;
         OS_RIGHT = 92;
+        NON_PRINTALBE_KEYS = [SHIFT, CTRL, ALT, OS_LEFT, OS_RIGHT, TAB, LEFT_ARROW, RIGHT_ARROW];
```
Comment (pdyraga) [inline dist/elemica-suggest.js]: `PRINTALBE` -> `PRINTABLE` ? :wink:

File: dist/elemica-suggest.js (hunk 1 above)
Comment (lukaszlenart) [reply dist/elemica-suggest.js]: No idea, it's what `npm run-script dist` produced. I didn't change any option related to IEs

File: dist/elemica-suggest.js (hunk 2 above)
Comment (lukaszlenart) [reply dist/elemica-suggest.js]: I think you can blame natives, I have just added `LEFT_ARROW` and `RIGHT_ARROW` ;-)

File: dist/elemica-suggest.js (hunk 1 above)
Comment (pdyraga) [reply dist/elemica-suggest.js]: Interesting... Let's trust coffee's compiler, we have no better option :wink:

File: dist/elemica-suggest.js (hunk 2 above)
Comment (lukaszlenart) [reply dist/elemica-suggest.js]: Fixed

**Comment by lukaszlenart** [comment]
Re-tested this new dist with mercury and everything looks good.

**Comment by pdyraga** [comment]
:+1:

**Pull request merged**